contrib/format-hdf5/src/main/java/org/apache/drill/exec/store/hdf5/writers/HDF5MapDataWriter.java

```diff
@@ -52,8 +52,8 @@ public HDF5MapDataWriter(IHDF5Reader reader, RowSetLoader columnWriter, String d
     } catch (Exception e) {
       throw UserException
         .dataReadError()
-        .message("Error writing Compound Field: ")
-        .addContext("Error", e.getMessage())
+        .message("Error writing Compound Field: {}", e.getMessage())
+        .addContext("Error: {}", e.getStackTrace())
```

Comment:
User execution will add stack trace itself, I believe, so adding it here can be omitted. Just remove `addContext` since looks all is covered in message.

Reply:
Done.

Reply:
Fixed that and found a few more string formats in UserExceptions.